config/nginx/conf.d/ubersystem.conf

```diff
@@ -0,0 +1,31 @@
+# redirect any HTTP traffic to HTTPS
+server {
+    listen 80;
+    return 301 https://$host$request_uri;
+}
+
+# serve up SSL content, proxy to the ubersystem app
+server {
+    listen 443 ssl;
+
+    ssl on;
+
+    ssl_certificate             /etc/nginx/certs/uber.crt;
+    ssl_certificate_key         /etc/nginx/certs/uber.key;
+    ssl_session_cache           shared:SSL:10m;
+    ssl_session_timeout         5m;
+    ssl_protocols               SSLv3 TLSv1 TLSv1.1 TLSv1.2;
+    ssl_ciphers                 HIGH:!aNULL:!MD5;
+    ssl_prefer_server_ciphers   on;
+
+    index                       index.html index.htm index.php;
+
+    location / {
+        return 301 https://$host/uber;
+    }
+
+    location /uber/ {
+        proxy_pass          http://web:8282/uber/;
+        proxy_read_timeout  90;
+    }
+}
```

Comment:
? question: I did notice that sideboard mounts some stuff on "/", is there any need to expose that on the public-facing proxy? there's some websocket stuff it looks like. I figure if we're a little leery of advertising that that stuff exists to the public, we could easily setup another 'server' directive here that handles just that stuff.

No need to expose it, sideboard mounts a couple of pages for convenience, like a page that shows you what plugins are installed and such.  Feel free to squash any of that and don't worry about proxying it.